docs/dev.rst

```diff
@@ -91,6 +91,17 @@ The Symbols Service webapp is at: http://localhost:3000
 How to
 ======
 
+How to set up a development container for VS Code
+-------------------------------------------------
+
+The repository contains configuration files to build a
+`development container <https://containers.dev/>`_ in the `.devcontainer`
+directory. If you have the "Dev Containers" extension installed in VS Code, you
+should be prompted whether you want to reopen the folder in a container on
+startup. You can also use the "Dev containers: Reopen in container" command
```

Comment:
Might be worth noting that they will likely need to restart VS Code after they install Dev Containers for the first time (as @relud mentioned they had issues opening the terminal for example on the first run until they did that).

Also even before that, might be worth mentioning that Tecken should be running locally first before any of this will work (might be obvious, but I prefer to overcommunicate generally).

Reply:
> Tecken should be running locally first before any of this will work

in particular I had to run `docker compose -f docker-compose.yml -f .devcontainer/docker-compose.yml build` before this worked for me, but I didn't actually have to have it running, that part worked automatically.

Reply:
Ah right, I was thinking about previously where we were connected to the `tecken-web` container that Tecken had to be running locally. In this case, this is a completely separate container that VS Code is starting up from the other Tecken containers, so Tecken doesn't need to be running (unless you want to run some integration or end to end tests that require communicating with other containers).

Reply:
It shouldn't actually be necessary to build the other containers manually first – at least that's my understanding how it is supposed to work, which could very well be wrong. I'll do a few tests before adding any further documentation.